apps/dashboard/src/contract-ui/tabs/overview/components/ContractChecklist.tsx

```diff
@@ -34,24 +32,19 @@ type Step = {
 export const ContractChecklist: React.FC<ContractChecklistProps> = ({
   contract,
 }) => {
-  const chain = useV5DashboardChain(contract.chainId);
-  const contractv5 = getContract({
-    address: contract.getAddress(),
-    chain: chain,
-    client: thirdwebClient,
-  });
+  const contractQuery = useContract(contract.address);
   const nftHref = useTabHref("nfts");
   const tokenHref = useTabHref("tokens");
   const accountsHref = useTabHref("accounts");
   const claimConditionsHref = useTabHref("claim-conditions");
 
-  const nfts = useNFTs(contract, { count: 1 });
-  const erc721Claimed = useClaimedNFTSupply(contract);
-  const claimConditions = useClaimConditions(contract);
-  const erc20Supply = useTokenSupply(contract);
-  const batchesToReveal = useBatchesToReveal(contract);
-  const accounts = useAccounts(contract);
-  const sharedMetadata = useSharedMetadata(contract);
+  const nfts = useNFTs(contractQuery.contract, { count: 1 });
+  const erc721Claimed = useClaimedNFTSupply(contractQuery.contract);
+  const claimConditions = useClaimConditions(contractQuery.contract);
+  const erc20Supply = useTokenSupply(contractQuery.contract);
+  const batchesToReveal = useBatchesToReveal(contractQuery.contract);
+  const accounts = useAccounts(contractQuery.contract);
+  const sharedMetadata = useSharedMetadata(contractQuery.contract);
```

Comment:
if ur doing this lets also move these to v5 then in this PR

Reply:
@jnsdls Can I stack them ser or do you want all of them in one PR?

Reply:
one pr ideally for this